nginx-webserver-02/nginx-webserver-01.dockerapp

```diff
@@ -0,0 +1,18 @@
+version: 1.0.0
+name: webserver-02
+description: Foundries.io webserver-02 example
+
+---
+
+version: '1.0'
+
+services:
+  nginx:
+    image: hub.foundries.io/munoz0raul/webserver-02:latest
```

Comment:
Here you have your own hub instead of following what you did at the first example, maybe change to <your factory> as well?